File: bm_rest_android/app/src/main/java/steffen/de/bm_rest_android/View/MainActivity.kt
package steffen.de.bm_rest_android.View

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import steffen.de.bm_rest_android.R
import steffen.de.bm_rest_android.ViewModel.MainActivityViewModel

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val viewModel = MainActivityViewModel(this)

        btn_get.setOnClickListener{
            viewModel.getPersons()
        }

        btn_post.setOnClickListener{
            viewModel.postPersons()
        }
    }
}

File: bm_rest_xam_platform/bm_rest_xam_android/MainActivity.cs
using Android.App;
using Android.Widget;
using Android.OS;
using Android.Support.V7.App;
using System;
using Shared.ViewModel;

namespace bm_rest_xam_android
{
    [Activity(Label = "@string/app_name", Theme = "@style/AppTheme", MainLauncher = true)]
    public class MainActivity : AppCompatActivity
    {
        private MainActivityViewModel MainActivityVM;
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            SetContentView(Resource.Layout.activity_main);

            MainActivityVM = new MainActivityViewModel();

            Button getBtn = FindViewById<Button>(Resource.Id.btn_get);
            getBtn.Click += OnGetClicked;

            Button postBtn = FindViewById<Button>(Resource.Id.btn_post);
            postBtn.Click += OnPostClicked;
        }

        private void OnGetClicked(object sender, EventArgs eventArgs)
        {
            MainActivityVM.GetPersons();
        }

        private void OnPostClicked(object sender, EventArgs eventArgs)
        {
            MainActivityVM.PostPersons();
        }
    }
}

